docs/content/latest/faq/general.md

```diff
@@ -1,12 +1,12 @@
 ---
-title: Product FAQ
-linkTitle: Product FAQ
-description: Product FAQ
+title: General FAQ
+linkTitle: General FAQ
+description: General FAQ
```

Comment:
FYI: I'm not familiar with any infra issues, if we change these titles / id below, if these are cached anywhere